src/main/java/techreborn/tiles/multiblock/TileImplosionCompressor.java

```diff
@@ -40,10 +42,12 @@
 import techreborn.client.container.builder.BuiltContainer;
 import techreborn.client.container.builder.ContainerBuilder;
 import techreborn.init.ModBlocks;
+import techreborn.tiles.TileRollingMachine;
```

Comment:
Why do you need this?

Reply:
forgot to remove it i think, i was trying stuff and i think i c/c in the wrong place

Reply:
You can make another commit to your branch with fix for unnecessary import. Your pull request will update automatically, as far as I know.